doc/readme/EXAMPLES.md

```diff
@@ -35,7 +35,7 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example: simple create another `*.sg` file.
+It is extremely simple to add a new example. Just create another `*.sg` file.
```

Comment:
I think this was meant to be: "It is extremely simple to add a new example: simply create another `*.sg` file."